doc/source/whatsnew/v0.23.0.txt

```diff
@@ -827,6 +827,7 @@ I/O
 - :class:`Interval` now supported in :func:`DataFrame.to_excel` for all Excel file types (:issue:`19242`)
 - :class:`Timedelta` now supported in :func:`DataFrame.to_excel` for xls file type (:issue:`19242`, :issue:`9155`)
 - Bug in :meth:`pandas.io.stata.StataReader.value_labels` raising an ``AttributeError`` when called on very old files. Now returns an empty dict (:issue:`19417`)
+- Bug in :func:`MultiIndex._format_native_types()`, meant to be a array of levels when level=1 instead of an array of tuples (:issue:`19589`)
```

Comment:
make this what a user would want to know. don't use internal functions, this is purely a ``.to_csv()`` with a MultiIndex incorrectly printing (from a users's perspective)

Reply:
Good point. a23252385e9de1a2af0832a82e168d825315d937